functions/Get-DbaPageFileSetting.ps1

```diff
@@ -0,0 +1,136 @@
+
+#>[CmdletBinding()]
+	param (
+		[Parameter(Mandatory = $false, Position = 0, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+		[Alias("cn", "host", "ServerInstance", "Server", "SqlServer")]
+		[object]$ComputerName,
+		[PSCredential] [System.Management.Automation.CredentialAttribute()]$Credential
+	)
+	BEGIN {}
+	PROCESS {
+        foreach ($Computer in $ComputerName)
+        {
+            Write-Verbose "Connecting to $Computer"
+			$reply = Resolve-DbaNetworkName -ComputerName $Computer -erroraction silentlycontinue
+            if ( $reply.ComputerName )
+            {
+                $computer = $reply.ComputerName
+            }
+            Write-Verbose "Connecting to $Computer via CIM (WSMan)"
+            $CompSys = Get-CimInstance -ComputerName $Computer -Query "SELECT * FROM win32_computersystem" -ErrorVariable $MyErr -Credential $Credential
+            if ( $CompSys )
+            {
+                Write-Verbose "Successfully retrieved PageFile information on $Computer via WSMan"
+                if ($CompSys.automaticmanagedpagefile -eq $False)
+                {
+                    $PF = Get-CimInstance -ComputerName $Computer -Query "SELECT * FROM win32_pagefile" # deprecated !
+                    $PFU = Get-CimInstance -ComputerName $Computer -Query "SELECT * FROM win32_pagefileUsage"
+                    $PFS = Get-CimInstance -ComputerName $Computer -Query "SELECT * FROM win32_pagefileSetting"
+                }
+            }
+			else
+            {
+				Write-Verbose "No WSMan connection to $Computer"
+                Write-Verbose "Getting computer information from server $Computer via CIM (DCOM)"
+			    $sessionoption = New-CimSessionOption -Protocol DCOM
+			    $CIMsession = New-CimSession -ComputerName $Computer -SessionOption $sessionoption -Credential $Credential
+                $CompSys = Get-CimInstance -CimSession $CIMsession -Query "SELECT * FROM win32_computersystem" -ErrorVariable $MyErr
+                if ( $CompSys )
+                {
+                    if ( $CompSys.automaticmanagedpagefile )
+                    {
+                        Write-Verbose "Successfully retrieved PageFile information on $Computer via DCom"
+                        if ($CompSys.automaticmanagedpagefile -eq $False)
+                        {
+                            $PF = Get-CimInstance -CimSession $CIMsession -Query "SELECT * FROM win32_pagefile" # deprecated !
+                            $PFU = Get-CimInstance -CimSession $CIMsession -Query "SELECT * FROM win32_pagefileUsage"
+                            $PFS = Get-CimInstance -CimSession $CIMsession -Query "SELECT * FROM win32_pagefileSetting"
+                        }
+                    }
+                    else
+                    {
+                    Write-Warning "$computer Operating System too old. No Pagefile information available."

```

Comment:
but when I change the settings so this become $false I receive this message but I don't receive it when it goes through the IF block. It is supposed to be like this?
If query throught DCOM the behavior shouldn't be the same? Or thought DCOM we can't access the properties?

IF block:
![image](https://cloud.githubusercontent.com/assets/19521315/22380184/6729a520-e4b4-11e6-9231-ccdd07ea73cb.png)

else block (without change settings):
![image](https://cloud.githubusercontent.com/assets/19521315/22380218/83024766-e4b4-11e6-88f3-4a05e599ea73.png)